### DISCUSSION TITLE
The larger the epoch, the larger the trained model?

### DISCUSSION THREAD

**Discussion by JiafengtTang** [open-edge-platform/anomalib#212] [Q&A]
Hi! When I was training the cflow model, the final trained model was too large and more than 4G. As the epoch increases, the model gets larger. After I set to save only model parameters, it did not solve the problem. Can you see what's going on? thank you very much.

**Comment by samet-akcay** [comment]
Hi @tjfstu, we'll try to reproduce the model size, and get back to you asap. Cheers
